resources/config/snemo/demonstrator/simulation/geant4_control/2.3/README.rst

```diff
@@ -0,0 +1,178 @@
+
+:Authors: François Mauger, Yves Lemière
+:Date:    2018-01-19
+
+.. contents::
+   :depth: 3
+..
+
+Presentation
+============
+
+This directory  contains the files  needed to run Geant4 simulation
+for the SuperNEMO demonstrator simulation.
+
+* Version is : ``2.2``
+
+This setup reuses a large part of the  ``2.1`` setup depends on:
+
+* the SuperNEMO demonstrator's geometry layout (``4.0``) and its variants.
+* the SuperNEMO vertex generation (``4.1``) and its variants.
+* the SuperNEMO primary event generation (``1.3``) and its variants.
+
+Files
+=====
+
+  * ``README.rst`` : This file.
+  * ``manager.conf``  :  The main  configuration  file for the
+  Bayeux/mctools Geant4 simulation engine.
+  * ``variants/``  : This directory contains configuration files
+    for variant support.
+
+Check the configuration
+=======================
+
+First make sure the Bayeux and Falaise softwares are installed and setup
+from a (Cadfael)brew shell:
+
+.. code:: sh
+
+   brewsh> which bxvariant_inspector
+   ...
+   brewsh> which bxg4_production
+   ...
+..
+
+Browse and edit primary event generation variant parameters and options
+===============================================================================
+
+1. From the build directory, generate documentation about variants:
+
+.. code:: sh
+
+   brewsh> bxvariant_inspector \
+          --datatools::resource-path="falaise@$(pwd)/BuildProducts/share/Falaise-3.0.0/resources" \
+          --variant-config "@falaise:config/snemo/demonstrator/simulation/geant4_control/2.2/variants/repository.conf" \
+	  --action doc > flg4ctrl.rst
+   brewsh> flg4ctrl.rst > flg4ctrl.html
+   brewsh> xdg-open flg4ctrl.html &
+   brewsh> rst2pdf flg4ctrl.rst
+   brewsh> xdg-open flg4ctrl.pdf &
+..
+or:
+
+.. code:: sh
+
+   brewsh> pandoc -t latex flg4ctrl.rst -o flg4ctrl.pdf
+..
+
+2. From the build directory, browse/edit the primary event generation variant and generate a variant profile::
+
+.. code:: sh
+
+   brewsh> bxvariant_inspector \
+	  --datatools::resource-path="falaise@$(pwd)/BuildProducts/share/Falaise-3.0.0/resources" \
+	  --variant-config "@falaise:config/snemo/demonstrator/simulation/geant4_control/2.2/variants/repository.conf" \
+          --variant-gui \
+	  --variant-store "myprofile.conf"
+..
+
+Run Geant4
+----------------------------------------------------------------------
+
+Run from  the Falaise build directory.  Here we **don't** use  the variant
+support but manually set vertex and particle generators associated to
+the default geometry:
+
+.. raw:: sh
+
+   brewsh> LD_LIBRARY_PATH="$(pwd)/BuildProducts/lib:${LD_LIBRARY_PATH}" \
+     bxg4_production \
+     --datatools::logging "warning" \
+     --datatools::resource-path "falaise@$(pwd)/BuildProducts/share/Falaise-3.0.0/resources" \
+     --logging-priority "debug" \
+     --load-dll Falaise \
+     --config @falaise:config/snemo/demonstrator/simulation/geant4_control/2.2/manager.conf \
```

Comment:
Not this tag (2.4), this carries on through the file...

Probably better to put the versions/tags as "placeholders" like `<FalaiseVersion>`, `<Tag>`, to minimise copy pasting work.